recipes/datrie/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "datrie" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7a11371cc2dbbad71d6dfef57ced6e8b384bb377eeb847c63d58f8dc8e8b2023
+
+build:
+  number: 0
+  skip: True  # [py27]
+  script: 
+    - cython src/datrie.pyx
```

Comment:
Why is this needed?

Reply:
Not sure. This what was done in the corresponding recipe for the windows anaconda package. I don't mind removing to see if it works without.

Reply:
Ah the [build script expects a C file]( https://github.com/pytries/datrie/blob/0.7.1/setup.py#L49-L54 ). Usually the build script would handle Cythonizing too, but I see we have to do it ourselves. Might be worth raising an issue upstream to ask them to handle this in `setup.py`.